server/chat-plugins/info.js

```diff
@@ -831,8 +839,15 @@ const commands = {
 
 		let targets = target.split(/[,+]/);
 		let sources = [];
-		let mod = Dex.mod(toId(targets[targets.length - 1])) || Dex;
-
+		let mod = Dex;
+		if (room && room.battle) {
+			let format = Dex.getFormat(room.battle.format);
+			mod = Dex.mod(format.mod);
+		}
+		if (targets[targets.length - 1] && toId(targets[targets.length - 1]) in Dex.dexes) {
+			mod = Dex.mod(toId(targets[targets.length - 1]));
+		}
+		//let mod = Dex.mod(toId(targets[targets.length - 1])) || roomMod;
```

Comment:
Remove this comment and your good to go!

Reply:
Done!